libs/wingsdk/src/shared-aws/queue.inflight.ts

```diff
@@ -54,14 +55,25 @@ export class QueueClient implements IQueueClient {
   }
 
   public async pop(): Promise<string | undefined> {
-    const command = new ReceiveMessageCommand({
+    const receiveCommand = new ReceiveMessageCommand({
       QueueUrl: this.queueUrl,
       MaxNumberOfMessages: 1,
     });
-    const data = await this.client.send(command);
+    const data = await this.client.send(receiveCommand);
     if (!data.Messages) {
       return undefined;
     }
-    return data.Messages[0].Body;
+
+    const message = data.Messages[0];
+
+    if (message.ReceiptHandle) {
```

Comment:
What does it mean if there's no receipt handle on the received message / we can't delete it? (curious if it should be an error of some kind, or if we should log something...)

Reply:
Actually, let me log it to start with